examples/tutorials/nb_python/ECCV_2020_Interactivity.py

```diff
@@ -0,0 +1,2093 @@
+# ---
+# jupyter:
+#   accelerator: GPU
+#   colab:
+#     collapsed_sections: []
+#     name: 'ECCV 2020: Interactivity.ipynb'
+#     provenance: []
+#     toc_visible: true
+#   jupytext:
+#     cell_metadata_filter: -all
+#     formats: nb_python//py:percent,colabs//ipynb
+#     notebook_metadata_filter: all
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.5.2
+#   kernelspec:
+#     display_name: Python 3
+#     name: python3
+# ---
+
+# %% [markdown]
+# #Habitat-sim Interactivity and Advanced Features
+#
+# This use-case driven tutorial covers Habitat-sim interactivity and advanced features, including:
+# - Adding new objects to a scene
+# - Kinematic object manipulation
+# - Physics simulation API
+# - Agent embodiment
+# - Object configuration via template libraries
+
+# %%
+# @title Installation { display-mode: "form" }
+# @markdown (double click to show code).
+
+# !curl -L https://raw.githubusercontent.com/facebookresearch/habitat-sim/master/examples/colab_utils/colab_install.sh | NIGHTLY=true bash -s
+# !wget -c http://dl.fbaipublicfiles.com/habitat/mp3d_example.zip && unzip -o mp3d_example.zip -d /content/habitat-sim/data/scene_datasets/mp3d/
+
+# %%
+# @title Path Setup and Imports { display-mode: "form" }
+# @markdown (double click to show code).
+
+import functools
```

Comment:
Why is functools imported? Is this an artifact of isort, I wonder. Not like it's do any harm though.

Reply:
I was using functools initially to support IPyWidgets' call back functions, but I figured out how to do what I wanted without it, so this can probably be removed.

Reply:
Yeah there is a mild isort bug with imports being moved between cell. There is a patch coming in the next version of JuPyText for it.